crates/router/src/connector/adyen/transformers.rs

```diff
@@ -915,7 +918,7 @@ fn get_line_items(item: &types::PaymentsAuthorizeRouterData) -> Vec<LineItem> {
             let line_item = LineItem {
                 amount_including_tax: Some(item.request.amount),
                 amount_excluding_tax: Some(item.request.amount),
-                description: None,
+                description: Some("NULL".to_string()),
```

Comment:
Why are we hardcoding the "NULL" value here?

Reply:
```suggestion
description: item.description,
```
@Sakilmostak Can we use payment request description as default instead of string "NULL"  ?

Reply:
It is the default case, when "none" was provided it was throwing 5xx error, there needed to be some value present in the description for it to work. Since it was a default case we could add anything, so I went with the current value